cmake/modules/hunter_fatal_error.cmake

```diff
@@ -8,7 +8,7 @@ include(hunter_wiki)
 
 function(hunter_fatal_error)
   cmake_parse_arguments(hunter "" "WIKI" "" "${ARGV}")
-  if(NOT hunter_WIKI)
```

Comment:
No, this one is correct. I guess the problem is in function calling `hunter_fatal_error` without `WIKI` argument. Like `hunter_fatal_error("Some message")`, should be `hunter_fatal_error("Some message" WIKI error.something)`.